CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.6.3 (2021-06-08)
+
+- Fixed to back to top component to hide the component if the `buttonHidden` option is enabled after the component is rendered. [#219](https://github.com/blackbaud/skyux-layout/pull/219)
```

Comment:
Is `component` correct here? Looking at the docs at https://developer.blackbaud.com/skyux-layout/docs/back-to-top?docs-active-tab=development, I assume this should be `directive`.

Reply:
Good catch. There is technically a component but the consumer uses the directive to show the component and the actual change was in the directive. I've fixed it.